pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +127,60 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterStorageClassChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
similar to https://github.com/kubernetes/kubernetes/pull/124961#discussion_r1615039425; if we can consider StorageClass update is not that frequent, it's not worthy for a fine but time-consuming filtering (using `PVCLister.PersistentVolumeClaims().Get()`).

Reply:
As a first step, we should make it minimum, at least until we establish some observability around QHint and could ensure this level of fine filtering doesn't impact a large cluster negatively (https://github.com/kubernetes/kubernetes/issues/124566).